src/main/java/frc/robot/commands/intake/StartIntake.java

```diff
@@ -40,6 +42,6 @@ public boolean isFinished() {
             timeoutCounter.increaseTimeoutCount();
             return true;
         }
-        return false;
+        return(SmartShuffleboard.getBoolean("Driver", "Gamepiece Collected", false));
```

Comment:
We use the feeder and Intake simultaniously almost all the time so I couldn't find a better way without having to remove the addRequirements in the Feeder commands

Reply:
In this particular case, I don't see anything wrong with the feeder _not_ being a subsystem requirement for this command, since all you need is the color sensor data. So as long as you don't pass the feeder into `addRequirements()`, it should be completely fine to use it without affecting other commands.